pkg/controller/toolchainstatus/toolchainstatus_controller.go

```diff
@@ -418,44 +418,65 @@ func compareAndAssignMemberStatuses(reqLogger logr.Logger, toolchainStatus *tool
 	return allOk
 }
 
-// replaceStatusConditions replaces Member status conditions with the new conditions
-func (r *ReconcileToolchainStatus) replaceStatusConditions(status *toolchainv1alpha1.ToolchainStatus, newConditions ...toolchainv1alpha1.Condition) error {
-	// the controller should always update at least the last updated timestamp of the status so the status should be updated regardless of whether
-	// any specific fields were updated. This way a problem with the controller can be indicated if the last updated timestamp was not updated.
+// updateStatusConditions updates ToolchainStatus status conditions with new conditions
+// the controller should always update at least the last updated timestamp of the status so the status should be updated
+// regardless of whether any specific fields were updated. This way a problem with the controller can be indicated if
+// the last updated timestamp was not updated.
+func (r *ReconcileToolchainStatus) updateStatusConditions(logger logr.Logger, status *toolchainv1alpha1.ToolchainStatus,
+	newConditions ...toolchainv1alpha1.Condition) error {
+
+	var updated bool
+
 	var conditionsWithTimestamps []toolchainv1alpha1.Condition
 	for _, newCondition := range newConditions {
 		condition := copyConditionWithTimestampSet(status, newCondition)
 		conditionsWithTimestamps = append(conditionsWithTimestamps, condition)
 	}
-	status.Status.Conditions = conditionsWithTimestamps
-	return r.client.Status().Update(context.TODO(), status)
+
+	status.Status.Conditions, updated = condition.AddOrUpdateStatusConditions(status.Status.Conditions, conditionsWithTimestamps...)
+	if !updated {
```

Comment:
we need to try to update the status every time - at this point we don't know if something has changed in the rest of the status - eg. number of users or memory usage in member clusters.
I'll fix that

Reply:
fixed in e6b496d